gulp-tasks/test.js

```diff
@@ -502,6 +503,19 @@ function testMarkdown(filename, contents, options) {
       });
     }
 
+    // Validate listed components
+    do {
```

Comment:
Instead of do-while, I have a preference for `.forEach` - see `// Check for uncommon tags` further down for the method I used. Not a hard requirement, but would like to keep consistency.